wheat_filtration/munge.py

```diff
@@ -5,6 +5,25 @@
 import nltk.data
 
 
+# make punctuation dictionary
+PUNC_DICT = str.maketrans('', '', string.punctuation)
+# unicode latin supplement block
+for key in range(128, 191):
+    PUNC_DICT[key] = None
+# unicode general punctuation block
+for key in range(8192, 8304):
+    PUNC_DICT[key] = None
+# replace hyphens with spaces
+PUNC_DICT[8212] = " "
```

Comment:
Any reason you don't have this in in the first maketrans line (e.g. by using unicode notation `"\u2010")? Also, do you need the other dash-like characters?

Sources:
https://docs.python.org/3/howto/unicode.html
http://jkorpela.fi/dashes.html#unidash